infra-gen2/backends/api/apiMultiAuth/amplify/backend.ts

```diff
@@ -1,11 +1,23 @@
-import { defineBackend } from '@aws-amplify/backend';
-import { auth } from './auth/resource';
-import { data } from './data/resource';
+import { defineBackend } from "@aws-amplify/backend";
+import { auth } from "./auth/resource";
+import { data } from "./data/resource";
 
 /**
  * @see https://docs.amplify.aws/react/build-a-backend/ to add storage, functions, and more
  */
-defineBackend({
+const backend = defineBackend({
   auth,
   data,
 });
+// extract L1 CfnUserPool resources
+const { cfnUserPool } = backend.auth.resources.cfnResources;
+// modify cfnUserPool policies directly
+cfnUserPool.policies = {
```

Comment:
Q: What is the default? Why do we need to change it? Can that change not be made in Gen 2?

Reply:
> What is the default?

8 Chars, 1 lower, 1 upper, 1 symbol

> Why do we need to change it?

To match the [current pattern](https://github.com/aws-amplify/amplify-flutter/blob/main/infra/lib/common.ts#L419-L427) and password generator methods we have. 

> Can that change not be made in Gen 2?

This is the [recommended](https://docs.amplify.aws/flutter/build-a-backend/auth/modify-resources-with-cdk/#override-cognito-userpool-password-policies) way when using Gen 2.

Reply:
The current generatePassword method used in integ tests inlcudes a UUID, an upper case character, and a symbol. That should always meet those requirements. Am I missing something?

https://github.com/aws-amplify/amplify-flutter/blob/7cca27606e9079a335ac7d6779f3438abff7cde5/packages/test/amplify_test/lib/src/mock_data.dart#L62-L65

Reply:
Ah, this function wasn't being called. Previously it was just a uuid for the password. Updated.